tool/generator/translator.dart

```diff
@@ -526,10 +526,12 @@ class Translator {
     for (final include in _includes) {
       final target = include.target;
       final includes = include.includes;
-      assert(_interfacelikes.containsKey(target));
```

Comment:
Aren't these still generated even though we don't emit them? Did you come across assertion failures?

Reply:
Hmm, yup, both assertions fired. This is something I didn't track down - why they're firing and what that means for the generated API.

I added a bit more diagnostics to see when they're triggering:


```
not in partial interface likes: PaintRenderingContext2D
not in partial interface likes: CSSPseudoElement
not in partial interface likes: NetworkInformation
not in partial interface likes: Bluetooth
not in partial interface likes: BluetoothDevice
not in partial interface likes: BluetoothRemoteGATTService
not in partial interface likes: BluetoothRemoteGATTCharacteristic
not in partial interface likes: GPUDevice
not in partial interface likes: GPUBuffer
not in partial interface likes: GPUTexture
not in partial interface likes: GPUTextureView
not in partial interface likes: GPUExternalTexture
not in partial interface likes: GPUSampler
not in partial interface likes: GPUBindGroupLayout
not in partial interface likes: GPUBindGroup
not in partial interface likes: GPUPipelineLayout
not in partial interface likes: GPUShaderModule
not in partial interface likes: GPUComputePipeline
not in partial interface likes: GPURenderPipeline
not in partial interface likes: GPUCommandBuffer
not in partial interface likes: GPUCommandEncoder
not in partial interface likes: GPUComputePassEncoder
not in partial interface likes: GPURenderPassEncoder
not in partial interface likes: GPURenderBundle
not in partial interface likes: GPURenderBundleEncoder
not in partial interface likes: GPUQueue
not in partial interface likes: GPUQuerySet


not in calculated mixins:HTMLAttributionSrcElementUtils [target=HTMLAnchorElement]
not in calculated mixins:HTMLAttributionSrcElementUtils [target=HTMLImageElement]
not in calculated mixins:HTMLAttributionSrcElementUtils [target=HTMLScriptElement]
not in calculated mixins:NavigatorBadge [target=Navigator]
not in calculated mixins:NavigatorBadge [target=WorkerNavigator]
not in calculated mixins:Region [target=Element]
not in calculated mixins:NavigatorDeviceMemory [target=Navigator]
not in calculated mixins:NavigatorDeviceMemory [target=WorkerNavigator]
not in calculated mixins:ARIAMixin [target=ElementInternals]
not in calculated mixins:NavigatorNetworkInformation [target=Navigator]
not in calculated mixins:NavigatorNetworkInformation [target=WorkerNavigator]
not in calculated mixins:HTMLSharedStorageWritableElementUtils [target=HTMLIFrameElement]
not in calculated mixins:HTMLSharedStorageWritableElementUtils [target=HTMLImageElement]
not in calculated mixins:NavigatorStorageBuckets [target=Navigator]
not in calculated mixins:NavigatorStorageBuckets [target=WorkerNavigator]
not in calculated mixins:NavigatorUA [target=Navigator]
not in calculated mixins:NavigatorUA [target=WorkerNavigator]
not in calculated mixins:ARIAMixin [target=Element]
not in calculated mixins:NavigatorAutomationInformation [target=Navigator]
not in calculated mixins:NavigatorGPU [target=Navigator]
not in calculated mixins:NavigatorGPU [target=WorkerNavigator]
not in calculated mixins:NavigatorML [target=Navigator]
not in calculated mixins:NavigatorML [target=WorkerNavigator]
```

Reply:
Ah I see, I think we're pruning libraries before we actually process them. I don't think it matters for the current purpose because we're allowlisting the stuff we need. Let's add a TODO just to make sure we keep track of this.